src/main/java/de/doubleslash/keeptime/Main.java

```diff
@@ -89,7 +87,7 @@ public void init() throws Exception {
       springContext = SpringApplication.run(Main.class);
       // TODO test if everywhere is used the same model
       model = springContext.getBean(Model.class);
-      controller = springContext.getBean(Controller.class);
+      model.setSpringContext(springContext);
```

Comment:
you included the dependency injection refactoring into this feature. should avoid mixing changes as it makes review harder and if one aspect is not ok, both features (here DI, Note reminder) are blocked from being merged.
Don't need to change it - just for later :)